File: PlantCareBuddy/PlantCareBuddy.Domain/Entities/CareEvent.cs
using PlantCareBuddy.Domain.Enums;
using System;

namespace PlantCareBuddy.Domain.Entities
{
    public class CareEvent
    {
        public Guid Id { get; set; }

        public Guid PlantId { get; set; }

        // Navigation property
        public virtual Plant Plant { get; set; }

        public CareEventType EventType { get; set; }
        public DateTime EventDate { get; set; }
        public string? Notes { get; set; }

        public string? BeforeImagePath { get; set; }
        public string? AfterImagePath { get; set; }
    }
}

File: client/src/models/CareEvent/careEvent.ts
import { CareEventType } from './careEventType';

export interface CareEvent {
  id: string;
  plantId: string;
  plantName: string;
  eventType: CareEventType;
  eventTypeName: string;
  eventDate: string;
  notes?: string;
  beforeImagePath?: string;
  afterImagePath?: string;
}